tests/app/index.js

```diff
@@ -2,101 +2,53 @@
  * @fileoverview Main generator tests
  * @author Kevin Partington
  */
-/* eslint no-invalid-this:0 */
+
 "use strict";
 
 //------------------------------------------------------------------------------
 // Requirements
 //------------------------------------------------------------------------------
 
 const assert = require("yeoman-assert"),
-    generators = require("yeoman-generator"),
     helpers = require("yeoman-test"),
-    path = require("path"),
-    sinon = require("sinon");
+    path = require("path");
 
 //------------------------------------------------------------------------------
 // Tests
 //------------------------------------------------------------------------------
 
-describe("ESLint Main Generator", () => {
-    const sandbox = sinon.createSandbox();
-
-    before(function(done) {
-        this.spy = sandbox.spy();
-
-        this.dummyGenerator = generators.Base.extend({
-            exec: this.spy
-        });
-
-        helpers.testDirectory(path.join(__dirname, "../../temp"), done);
-    });
-
-    beforeEach(function() {
-        this.spy.resetHistory();
-    });
+const PATH_APP_GENERATOR = path.join(__dirname, "..", "..", "app");
```

Comment:
We’ve typically suffixed paths with the word “path” rather than prefixed. 
```suggestion
const APP_GENERATOR_PATH = path.join(__dirname, "..", "..", "app");
```

Reply:
Fixed variable name.